lib/usd/CMakeLists.txt

```diff
@@ -1 +1,10 @@
 add_subdirectory(translators)
+
+if(CMAKE_WANT_QT_BUILD)
+    if(DEFINED QT_LOCATION)
+        message(STATUS "Building with Qt UI features enabled.")
+        add_subdirectory(ui)
+    else()
+        message(STATUS "QT_LOCATION not set. Building UI features will be disabled.")
+    endif()
+endif()
```

Comment:
When the new cmake option is on (enabled by default) and we got a QT_LOCATION passed to build, we'll add the new "ui" subdir.